src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 0.3s;
```

```suggestion
    transition: transform, color 0.3s;
```
Потрібно конкретно прописувати, для яких стилів застосовується transition